auth_totp/models/res_users.py

```diff
@@ -13,6 +13,12 @@
 class ResUsers(models.Model):
     _inherit = 'res.users'
 
+    @classmethod
+    def _build_model(cls, pool, cr):
+        ModelCls = super(ResUsers, cls)._build_model(pool, cr)
+        ModelCls.SELF_WRITEABLE_FIELDS += ['mfa_enabled', 'authenticator_ids']
+        return ModelCls
+
```

Comment:
This depends on an implementation detail and not on the API.
What is the point? Just make the fields writeable?